docs/zh/connector-v2/source/GraphQL.md

```diff
@@ -0,0 +1,188 @@
+import ChangeLog from '../changelog/connector-graphql.md';
+
+# GraphQL
+
+> GraphQL source connector
```

Comment:
Please use `zh`

Reply:
I'd like to keep this file for first-time contributors